services/common/src/components/projectSummary/DocumentUpload.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import FormWrapper from "@mds/common/components/forms/FormWrapper";
+import { FORM } from "@mds/common/constants/forms";
+import { ReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
+import * as MOCK from "@mds/common/tests/mocks/dataMocks";
+import { PERMITS, PROJECTS, STATIC_CONTENT } from "@mds/common/constants/reducerTypes";
+import DocumentUpload from "./DocumentUpload";
+
+const initialState = {
+  [PROJECTS]: {
+    projectSummary: MOCK.PROJECT_SUMMARY,
+  },
+  [PERMITS]: {
+    permits: MOCK.PERMITS,
+  },
+  [STATIC_CONTENT]: {
+    projectSummaryPermitTypes: MOCK.BULK_STATIC_CONTENT_RESPONSE.projectSummaryPermitTypes,
+    projectSummaryAuthorizationTypes:
+      MOCK.BULK_STATIC_CONTENT_RESPONSE.projectSummaryAuthorizationTypes,
+  },
+};
+
+describe("DocumentUpload", () => {
+  it.skip("renders properly", () => {
```

Comment:
Oops. I _knew_ I committed at least one skipped test in there. Try to get it working.